src/utils/dateUtils.ts

```diff
@@ -0,0 +1,18 @@
+import dayjs from 'dayjs';
+
+export const secondsToDate = (seconds: number) => {
+  let year: string | number = dayjs(seconds).year();
+  let day: string | number = dayjs(seconds).date();
+  let month: string | number = dayjs(seconds).month() + 1;
+  day = day < 10 ? '0' + day : day;
+  month = month < 10 ? '0' + month : month;
+  return day + '.' + month + '.' + year;
```

Comment:
I'm thinking dayjs will have better solution for this ;)

Reply:
Not done

Reply:
done